tests/units/cfixertest.cpp

```diff
@@ -20,3 +22,9 @@ ExampleTest::~ExampleTest() {
 void ExampleTest::test() {
     QVERIFY(true);
```

Comment:
You forget invoke your method testFoo in this function.

Reply:
testFoo();

Reply:
Done!